front/src/views/EditCustomProduct/EditCustomProductView.tsx

```diff
@@ -105,7 +110,7 @@ function EditCustomProductFormContainer() {
           gender: editProductFormOutput.gender?.value as NonNullProductGender,
           price: editProductFormOutput.price,
           inShop: editProductFormOutput.inShop,
-          comment: editProductFormOutput.comment,
+          comment: editProductFormOutput.comment || "",
```

Comment:
not sure why you do this, but it actually causes empty strings to be saved in the db which I would not like to do.

Reply:
@HaGuesto please check the conversation that Felipe and I had. 
This is necessary in order to reset a comment

Reply:
Also, saving an empty string as product or box comment is the default in dropapp